### DISCUSSION TITLE
sn-scriptsync and background scripts

### DISCUSSION THREAD

**Discussion by codaroma** [arnoudkooi/sn-scriptsync#98] [Q&A] [answered]
I am trying to get the sn-scriptsync and background scripts integration working. I am not sure what exactly is supposed to happen? And I am not sure if what is happening is how it is suppose to work? I can sync the code from the background script to vscode but when I update/save the code in vscode is it suppose to update/sync back into ServiceNow? I am using firefox 126.0 and sn utils 8.1.2.2

**Comment by arnoudkooi** [comment, marked as answer]
That is it, no (more) syncing back, you can now execute it inside VS Code via the context menu
